examples/get_iss_breast_data.py

```diff
@@ -0,0 +1,139 @@
+        ch = [2, 1, 3, 0]
+        ch_dict = dict(zip(ch, ch_str))
+        return ch_dict
+
+    @property
+    def hyb_dict(self):
+        hyb_str = ['1st', '2nd', '3rd', '4th']
+        return dict(zip(range(4), hyb_str))
+
+    def get_tile(self, fov: int, hyb: int, ch: int, z: int) -> FetchedTile:
+        filename = 'slideA_' + str(fov + 1) + '_' + \
+                   self.hyb_dict[hyb] + '_' + self.ch_dict[ch] + '.TIF'
+        file_path = os.path.join(self.input_dir, filename)
+        return IssCroppedBreastTile(file_path)
+
+
+class ISSCroppedBreastAuxTileFetcher(TileFetcher):
+    def __init__(self, input_dir, aux_type):
+        self.input_dir = input_dir
+        self.aux_type = aux_type
+
+    def get_tile(self, fov: int, hyb: int, ch: int, z: int) -> FetchedTile:
+        if self.aux_type == 'nuclei':
+            filename = 'slideA_' + str(fov + 1) + '_DO_' + 'DAPI.TIF'
+        elif self.aux_type == 'dots':
+            filename = 'slideA_' + str(fov + 1) + '_DO_' + 'Cy3.TIF'
+        else:
+            msg = 'invalid aux type: {}'.format(self.aux_type)
+            msg += ' expected either nuclei or dots'
+            raise ValueError(msg)
+
+        file_path = os.path.join(self.input_dir, filename)
+
+        return IssCroppedBreastTile(file_path)
+
+
+def format_data(input_dir, output_dir):
+    if not input_dir.endswith("/"):
+        input_dir += "/"
+
+    if not output_dir.endswith("/"):
+        output_dir += "/"
+
+    def add_codebook(experiment_json_doc):
+        experiment_json_doc['codebook'] = "codebook.json"
+        return experiment_json_doc
+
+    num_fovs = 16
+
+    hyb_dimensions = {
+        Indices.ROUND: 4,
+        Indices.CH: 4,
+        Indices.Z: 1,
+    }
+
+    aux_name_to_dimensions = {
+        'nuclei': {
+            Indices.ROUND: 1,
+            Indices.CH: 1,
+            Indices.Z: 1,
+        },
+        'dots': {
+            Indices.ROUND: 1,
+            Indices.CH: 1,
+            Indices.Z: 1,
+        }
+    }
+
+    write_experiment_json(output_dir,
+                          num_fovs,
+                          hyb_dimensions,
+                          aux_name_to_dimensions,
+                          primary_tile_fetcher=ISSCroppedBreastPrimaryTileFetcher(input_dir),
+                          aux_tile_fetcher={
+                              'nuclei': ISSCroppedBreastAuxTileFetcher(input_dir, 'nuclei'),
+                              'dots': ISSCroppedBreastAuxTileFetcher(input_dir, 'dots'),
+                          },
+                          postprocess_func=add_codebook,
+                          default_shape=SHAPE
+                          )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input_dir", type=FsExistsType())
+    parser.add_argument("output_dir", type=FsExistsType())
+
+    args = parser.parse_args()
+    s3_bucket = "s3://czi.starfish.data.public/browse/raw/20180820/iss_breast/"
```

Comment:
i don't get this.  who's going to do this?

Reply:
ambrose is going to do this. josh is going to read this code. this is also an example to help collaborators understand how to convert their data.

Reply:
I think this message is confusing. i would just have the `--help` output tell people where to get the data.

Rationale: the only time this message prints is if the command is running, and to do that, you kind of need the data already.

Reply:
👍